### DISCUSSION TITLE
Facebook Icon gone - And one Link is not working

### DISCUSSION THREAD

**Discussion by root9191** [JulianPrieber/littlelink-custom#122] [Issues]
Hello,
i am using the actual Version of Littlelink Custom and for some reason i don't see the Facebook Icon anymore.
Also one Link is not working - after clicking it, i get an 404 Error. The Link is working when i open it on any Browser.
Does anyone know what to do?

Edit:
OK so i found out, that ublock was blocking the Facebook Icon. 
And i also found out, why the Link is not working. The Link has an "?" in the URL and that's why Littlelink Custom can not redirect it.
Is there any Solution for that?

**Comment by JulianPrieber** [comment]
Hi @root9191,

Thank you for pointing that out, I never noticed that 😵‍💫. With our current system, this is probably not fixable.

Your best bet is probably to use a link shortener like [bit.ly](https://bitly.com/).

**Reply by root9191** [reply]
That's a good workaround i didn't think of that 😅
Thank you.
